### ISSUE TITLE
Security breach: Showing reCaptcha secretKey in GraphQL Public Schema response

### ISSUE THREAD

**Issue by denisyilmaz** [verbb/formie#1187]
### Describe the bug

When querying for the integration settings of a form via GraphQL the settings string contains the secretKey of the reCaptcha settings.

### Query
```gql
query {
  formieForm {
    id
    title
    settings {
      integrations {
        name
        handle
        enabled
        settings
      }
    }
  }
}
```

### Response  
_moved the fields to the start of the string for better visibility_
```json
{
    "id": "2779",
    "siteId": 1,
    "title": "Test",
    "slug": null,
    "settings": {
      "integrations": [
        {
          "name": null,
          "handle": "recaptcha",
          "enabled": true,
          "settings": "{\"id\":null,\"secretKey\":\"<the-secret-key>\",\"siteKey\":\"<the-site-key>\",\"name\":null,\"handle\":\"recaptcha\",\"tokenId\":null,\"ipAddress\":\"\",\"showAllPages\":false,\"spamReason\":null,\"saveSpam\":true,\"size\":\"normal\",\"theme\":\"light\",\"badge\":\"bottomright\",\"language\":\"en\",\"minScore\":0.5,\"projectId\":null,\"scriptLoadingMethod\":\"asyncDefer\"}"
        },
        {…}
      ]
    }
}
```

As by the [reCaptcha Develops Guide](https://developers.google.com/recaptcha/intro#recaptcha-overview) the secret key should kept safe.

I don't see the overall need to query the `secretKey` via GraphQL in any way as this is used for the frontend represantation of the forms, the validation should happen on the Craft CMS server in any way.  
Maybe (for reasons i dont see) the `secretKey` is needed in some scenarios, it should have its own "GraphQL Schema Rule" like `View Form Setting secrets"

<img width="223" alt="image" src="https://user-images.githubusercontent.com/4189607/202897513-2dff12c2-cbe9-4d13-9522-e9e5a52cd298.png">

### Steps to reproduce

1. Add new Form
2. Configure reCaptcha integration
3. Query form with example above.

### Form settings

- Headless integration of Formie via Graphql

### Craft CMS version

4.3.3

### Plugin version

2.0.18

### Multi-site?

No

**Comment by engram-design** [comment]
Yeah, it's pretty arguable that no other integrations should return any of their settings to the client. You really only care about captchas, which have client-side implications (Address Provider integrations are different, as they're fields).

As such, we now take a more heavy-handed approach that no other integration other than reCAPTCHA and hCaptcha (and any more similar integrations in the future) return data from this query now.

Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.19](https://github.com/verbb/formie/releases/tag/2.0.19)
